BraveShared/Analytics/DAU.swift

```diff
@@ -86,6 +94,7 @@ public class DAU {
             Preferences.DAU.lastLaunchInfo.value = paramsAndPrefs.lastLaunchInfoPreference
             
             Preferences.DAU.lastPingFirstMonday.value = paramsAndPrefs.lastPingFirstMondayPreference
+            self.processingPing = false
```

Comment:
You need to set self.processingPing = false in above `if let e = error {` also to complete the check.

Reply:
yikes, thx